test/spectests/spectests.cpp

```diff
@@ -525,7 +522,7 @@ class test_runner
             return false;
         }
 
-        const auto expected_value = read_floating_point_value<T>(expected);
+        const auto expected_value = read_value(expected)->value;  // TODO: Type ignored.
```

Comment:
Is this comment still relevant? (second commits removes the same comment for integers, but the code is quite similar)

Reply:
Yes. read_value() produces type and value, but type is discarded here.